docs/release-notes/RELEASE-NOTES-0.12.0.md

```diff
@@ -23,11 +23,13 @@ The instructions below describe:
 
 ## Retrieve the software for running a node
 
-You can run a node by using either:
-* The published [release binary][this-release] corresponding to your platform; or
-* The published [Docker image `aeternity/epoch`][docker].
+You can run a node by either:
+* Install the published [release binary][this-release] corresponding to your platform; or
+* Run the published [Docker image `aeternity/epoch`][docker]; or
+* [Build it from source][build].
```

Comment:
"it" meaning the published release binary or published Docker image? Please remove usage of pronoun "it".

Reply:
improved in 24e70eb2